src/App.js

```diff
@@ -1,16 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import './components/ChatEntry';
+import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+
+  const [messages, setMessages] = useState(chatMessages);
```

Comment:
👍 We need a piece of state to track changes to our messages, initialized from the json file we loaded.